client/src/pages/Statistics.js

```diff
@@ -1,53 +1,66 @@
 import axios from 'axios';
 import React, { useState, useEffect } from 'react';
+import ProgressBar from '../components/ProgressBar';
 
 function Statistics() {
-    // eslint-disable-next-line
-    const [loading, setLoading] = useState(true);
-    const [data, setData] = useState([]);
+    const [progress, setProgress] = useState(0);
+    const [averageReadingTime, setAverageReadingTime] = useState(null);
+    const [count, setBooksCompleted] = useState(0);
+    const [readingGoal, setReadingGoal] = useState(0); // New state for reading goal
+    const yearlyReadingGoal = readingGoal; // Use the fetched reading goal
 
     useEffect(() => {
-        const fetchData = async () => {
-            console.log("Fetching data....")
-            setLoading(true);
-            try {
-                const { data: response } = await axios.get('http://localhost:5000/books');
-                setData(response);
-            } catch (error) {
-                console.error(error.message);
-            }
-            setLoading(false);
-        }
+        // Fetch user data including reading goal
+        axios.get('http://localhost:5000/users/1')
```

Comment:
Will probably want to change all of these "localhost" to reflect some HOST variable in an environment file or some other way. Not a detriment to *this* PR, but it may prove annoying in the future.

Reply:
Added a proxy configuration in the package.json file by following [these instructions](https://create-react-app.dev/docs/proxying-api-requests-in-development/)

However, when we deploy this app, we'll probably have to do some more configuring with the public development host on an .env.development file https://create-react-app.dev/docs/proxying-api-requests-in-development/#invalid-host-header-errors-after-configuring-proxy